packages/palette/src/helpers/typography.ts

```diff
@@ -0,0 +1,7 @@
+import { endash } from "../elements/Typography"
+
+/**
+ * A helper func to create a range string out of two strings, by putting an endash between them.
+ */
+export const range = (left: string, right: string): string =>
+  `${left} ${endash} ${right}`
```

Comment:
Totally up to your discretion, but maybe this can live directly in the `Typography` component? Then, later, if we find a lot of util-like typography functions being used we can extract into a file.

Reply:
I kinda like to have them separate, seeing the rest of the structure of the repo. I find it harder to split a big file than having more small files 🤔.